src/viam/examples/modules/complex/gizmo/api.cpp

```diff
@@ -55,8 +57,8 @@ Gizmo::Gizmo(std::string name) : Component(std::move(name)){};
 
 /* Gizmo server methods */
 
-GizmoServer::GizmoServer() : ResourceServer(std::make_shared<ResourceManager>()){};
-GizmoServer::GizmoServer(std::shared_ptr<ResourceManager> manager) : ResourceServer(manager){};
+GizmoServer::GizmoServer(std::shared_ptr<ResourceManager> manager)
+    : ResourceServer(std::move(manager)){};
```

Comment:
Per PR description, I've removed all `[TypeOfResource]Server` constructors that took in no `ResourceManager`, made the remaining constructor `explicit`, and I've `std::move`d the manager (probably not necessary, but apparently a good habit).